virttest/utils_test/__init__.py

```diff
@@ -402,6 +402,12 @@ def run_image_copy(test, params, env):
     2) Check the existence of source image
     3) If it exists, copy the image from NFS
 
+    Note about 'source_image_name' parameter.  'image_copy' test is used for
+    copying existing/prestine image for usage in further tests.
```

Comment:
Yep, it should be pristine.  @Andrei-Stepanov can you force push a fix and this seems good to go.